butte/src/parser.rs

```diff
@@ -888,19 +937,31 @@ rpc_service Greeter {
     }
 }
 
+#[cfg(test)]
+mod dotted_ident_tests {
+    use super::*;
+
+    #[test]
+    fn test_simple_dotted_ident() {
+        let result = dotted_ident("a.b");
+        let expected = DottedIdent::from(vec![Ident::from("a"), Ident::from("b")]);
+        assert_successful_parse!(result, expected);
+    }
+}
+
 pub fn rpc_method(input: &str) -> IResult<&str, RpcMethod> {
     map(
         tuple((
             terminated(ident, comment_or_space0),
             delimited(
                 left_paren,
-                delimited(comment_or_space0, ident, comment_or_space0),
+                delimited(comment_or_space0, dotted_ident, comment_or_space0),
```

Comment:
Is "dotted" just a something with a qualified path?

Reply:
Yep. I tried to roll this all into `Ident` but it forced me to start using `String` in a bunch of places so I abandoned that approach.

Reply:
Actually this separation seems to be required since `a.b.c` isn't valid in certain contexts like

```
rpc_service a.b.c {
    ...
}
```